primer/brand
# Add size prop to Heading component API

Closes https://github.com/primer/brand/issues/101

## Summary
Adds a `size` prop to the `Heading` component, which lets you override the visual appearance of a heading without affecting the html element being used.

**Why?**

The default styles and tight coupling between html tag and rendered text size is too restrictive. This is an escape-hatch for situations where we need granular control over the visual output.



## What should reviewers focus on?

- Try the examples out ([link](https://primer-4f6ab6165d-26139705.drafts.github.io/components/Heading))

## Steps to test:



1. Open Docs and review the examples and prop table output
2. Go to story in Storybook and verify the examples

## Supporting resources (related issues, external links, etc):

- https://github.com/primer/brand/issues/101
-

## Contributor checklist:


## Reviewer checklist:


## Screenshots:
![storybook](https://user-images.githubusercontent.com/13340707/196201847-cb6524dc-9bf7-45df-9c9a-eee4a92e7b9e.png)
